Restaurant page code review.

dipbazz opened the pull request:
In this pull request, the features added are:

- User can view the homepage.
- User can navigate to the menu link and view a list of all menus item.
- User can navigate to the about and contact section.

For every navigation link, I have created a module with specific HTML content to it. Different module present are:

- `pageLoad.js` contains the header with nav links for the site.
- `homePage.js` contains the information displayed on the landing page for home.
- `menuPage.js` contains the list of menus on a card with information about the food item and prices.
- `aboutPage.js` contains the information about the project.
- `contactPage.js` contains the contact form.
- `index.js` loads all the different modules and also contains the logic for the navigation menu link to change on click.

dipbazz committed:
Add project setup files like linters and so on

dipbazz committed:
Add src and dist directory with required files

configured webpack to build js files

dipbazz committed:
Add different module that contains page specific

pageLoad.js module contains the header element on load.
homePage.js module contains homepage content that's tabable
menuPage.js module contains menu content that's tabable
aboutPage.js module contains about content that's tabable
contactPage.js module contains contact content that's tabable

dipbazz committed:
Update index.html with menu list items ui

menu list display in a card with image and menu informations

dipbazz committed:
Add layout for menu, about and contact page

- create card to display menu items in a list.
- made the content in the center with max width to fit on center
  for every page.
- add images for menu items.

dipbazz committed:
Update readme file with information about project.

dipbazz committed:
Fix eslint errors and update packge.json file

addod19 commented on package.json:
```diff
@@ -0,0 +1,34 @@
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.24.4",
+    "webpack-cli": "^4.5.0"
+  },
```
Kindly add the following to the devDependencies ` webpack, webpack-cli, webpack-dev-server`

addod19 commented on package.json:
```diff
@@ -0,0 +1,34 @@
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Let us improve our script so we do away with the warnings when we run the command `npm run build/watch`. Use this instead 
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

addod19 reviewed (changes requested):
## Status: Require changes :recycle: :traffic_light: 

Hi @dipbazz, congratulations on this project :muscle: Awesome job :+1: The following requirements have been met:
- README is professional :100: 
- PR is detailed with information :1st_place_medal: 
- Commit messages are easy to understand even for non-techincal guys
- Application works as expected :+1: 
- Great use of ES6 features :smile: 

However, we need to fix the following comments so we get our project approved :+1: Kindly check the comments below:
- Kindly install the following devDependencies; webpack, webpack-cli, webpack-dev-server
- Also let us improve our script so we can run the npm commands without any warnings

Once these are fixed, kindly submit another code review request :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

dipbazz committed:
Add webpck-dev-server package to use live reload

Update package.json scripts to resolve the warning and
add new script.

update webpack.config.js with devServer config to look for
index file.

teekaytech commented on dist/style.css:
```diff
@@ -0,0 +1,219 @@
+  height: 3px;
+  position: absolute;
+  bottom: 0;
+  /* left: 50%;
+  transform: translateX(-50%); */
```
Kindly remove all commented blocks/lines of code.

teekaytech reviewed (approved):
## Status: Approved :green_circle: 

### Congratulations @dipbazz :tada: 

**This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

**Comment(s)**
- Kindly check my inline comment below, do ensure that the commented lines of code are deleted before you merge.

Again, congratulations!
Do not forget to fill the project completion form.

**Good luck with your next task.**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

dipbazz committed:
Remove commented code from style.css
